batch-setup/make_rawr_tiles.py

```diff
@@ -75,21 +77,27 @@ def missing_jobs(missing_bucket, rawr_bucket, date_prefix, region, config,
 
     tiles = missing_tiles(
         missing_bucket, rawr_bucket, date_prefix, region, key_format_type,
-        config, group_by_zoom, tile_coords_generator)
+        config, group_by_zoom, tile_coords_generator, tile_verifier)
 
     # the rawr tiles of `group_by_zoom` is actually built by AWS batch jobs of
     # `queue_zoom` so we need to do a zoomTo here to find the corresponding jobs
-    jobs = set(coord.zoomTo(queue_zoom).container() for coord in tiles)
+    job_coords = {coord.zoomTo(queue_zoom).container() for coord in tiles}
 
-    print("[make_rawr_tiles] Missing %d tiles (%d jobs)" % (len(tiles), len(jobs)))
+    if bool(tile_coords_generator):
+        print("[make_rawr_tiles] Going to rebuild %d tiles by (%d jobs)" % (len(tiles), len(job_coords)))
+    else:
+        print("[make_rawr_tiles] Missing %d tiles (%d jobs)" % (len(tiles), len(job_coords)))
 
     tmpdir = tempfile.mkdtemp()
     try:
-        missing_file = os.path.join(tmpdir, "missing_jobs.txt")
+        missing_file = os.path.join('/home/ec2-user/missingfile', "missing_jobs.txt")
```

Comment:
Can this go back to tmpdir? If not, can you add a comment explaining why?

Reply:
oh yes, forgot to revert; fixed in the next commit

Reply:
+1